lib/osf-components/addon/components/registries/schema-block-renderer/read-only/response/template.hbs

```diff
@@ -4,8 +4,10 @@
 >
     {{~if this.readOnlyValue this.readOnlyValue (t 'osf-components.registries.schema-block-renderer/read-only/response.noResponse')~}}
 </p>
-<ValidationErrors
-    local-class='ValidationErrors'
-    @changeset={{@changeset}}
-    @key={{@schemaBlock.registrationResponseKey}}
-/>
\ No newline at end of file
+{{#if @changeset}}
```

Comment:
Same here. But if it's some minor bug discovered during the process of converting i18n, it's okay to leave in.

Reply:
ditto ⬆️